packages/AlgernonTests.package/ALGTestUI.class/instance/testAlgernonPositioningWindowIsSameSizeAfterReopening.st

```diff
@@ -0,0 +1,8 @@
+testing
+testAlgernonPositioningWindowIsSameSizeAfterReopening
+	| point oldPosition|
+	point := 0 @ 0.
+	oldPosition := self algerInstance position.
+	self algerInstance lastWorldBounds: World bounds corner - point.
```

Comment:
Please look at the errors squeak gives you. They will tell you what has done wrong. If you did that, you'd easily see that the errors are because you removed `lastWorldBounds` from ALGCore, but still call it from the tests.